packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -0,0 +1,112 @@
+/* eslint-disable global-require, import/no-dynamic-require, react/no-unused-prop-types */
```

Comment:
no-dynamic-require and global-require causes lint here, but is needed to load the appropriate translations. https://github.com/cerner/terra-core/pull/191/files#diff-c554f9bb4d56617bb2a5b86d37b215f2R85

Since I am passing all the props straight to the banner, react/no-unused-prop-types is throwing a lint warning.